.github/workflows/build-and-test.yml

```
@@ -12,19 +12,49 @@ on:
       - epic/*
     paths:
       - '.github/workflows/build-and-test.yml'
+      - '.github/workflows/scripts'
       - '.github/actions'
       - '**.rs'
       - '**.toml'
+      - 'bindings/wasm/'
+      - '!bindings/wasm/**.md'
 
 env:
   RUST_BACKTRACE: full
+  CARGO_INCREMENTAL: 0 # disabled to reduce target cache size and improve sccache (https://github.com/mozilla/sccache#known-caveats)
   SCCACHE_CACHE_SIZE: 2G
   SCCACHE_IDLE_TIMEOUT: 0
   # SCCACHE_RECACHE: 1 # uncomment to clear sccache cache, then re-comment
 
 jobs:
+
+  determine-modification:
```

Nit: name suggestion.
```suggestion
  check-core-modified:
```

---

I see this job checking for more modifications, when we add more bindings. Can we live with keeping it generic?

---

I guess so, it's just a bit hard to grok what it's for by the name alone. I'm fine leaving it as-is.

---

maybe just `check-for-modification` or something?

---

@cycraig what do you think?

---

Oh! Yeah I'm fine with either `determine-modification` or `check-for-modification`. The latter might be more immediately descriptive?

---

went with `check-for-modification`